### ISSUE TITLE
Clementine will not add new music to library

### ISSUE THREAD

**Issue by mdj6248** [clementine-player/Clementine#6481]
### Before posting

Please follow the steps below and check the boxes with [x] once you did the step.

- [ x] I checked the issue tracker for similar issues
- [x ] I checked the [changelog](https://github.com/clementine-player/Clementine/blob/master/Changelog) if the issue is already resolved
- [x ] I tried the latest Clementine build from [here](https://builds.clementine-player.org/)

### System information

Please provide information about your system and the version of Clementine used.

- Operating System: Ubuntu 19.10 (Eoan)
- Clementine version: Clementine 1.3.1-967-g922e10bc4

### Expected behaviour / actual behaviour
My library is organized as `/music/<artist>/<album>/songs `(mp3 or flac), with `/music` defined as the location in preferences.
I recently purchased a copy of the Cake album "Fashion Nugget". I added it to other Cake albums I have as the directory `"/music/Cake/Fashion Nugget"` expecting Clementine would detect it and add it to my library.

Clementine will not add it to my library. I can select the files in the directory and add them to a playlist and they play fine.

I have tried:
1) Deleting the directory and recreating it.
2) touching the directory and doing an "Update changed library folders".
3) Doing a full rescan.
4) Exiting and restarting Clementine.
5) Deleting the Clementine database and letting it rebuild it from scratch (with the files in place).

I have also checked and the owner/group and permissions for the directory and the files inside it are the same as other directories that were automatically picked up by Clementine.

**Comment by jbroadus** [comment]
I feel like I saw something like this at some point, and wish I could remember what I did.

A few questions:
- If you run with --verbose, are there any error messages that might be relevant?
- Is /music on local media, or is it a network mount?
- If you copy one of the music files to a directory with music that did scan correctly, will Clementine pick it up?

**Comment by mdj6248** [comment]
It's on a network mount. I solved the problem, actually, and should have closed this issue. There was still a more subtle error - for some reason, the tracks in question were being added with incorrect artist names and I wasn't finding them until I searched by *track* name. I don't know why that happened, and it's probably a lot more difficult to diagnose. I'll see if I can capture info if it happens again.
